app/components/case-studies-components/CaseStudyCard.tsx

```diff
@@ -16,8 +20,8 @@ const CaseStudyCard = ({ mainProject }: CaseStudyHeaderProps) => {
             backgroundColor: mainProject?.backgroundColour,
           }}
         >
-          {mainProject?.desktopImage && (
-            <LaptopMockup imageSrc={mainProject?.desktopImage} />
+          {mainProject?.desktopImage.image && (
+            <LaptopMockup imageSrc={mainProject?.desktopImage.image} />
```

Comment:
Why not use the Image component that Nextjs provides? You could use `src` with Image if that's what you're trying to accomplish here. 

```
import Image from 'next/image'
import profilePic from './me.png'
 
export default function Page() {
  return (
    <Image
      src={profilePic}
      alt="Picture of the author"
      ...
    />
  )
}
```

You can read more about it here: https://nextjs.org/docs/app/building-your-application/optimizing/images

Reply:
Hi @cdsalty I have made LaptopMockup into a reusable component in which there is an Image tag amongst over things required to style the component correctly. I found it easiest to pass the source to the reusable component in this case. Do you think that's ok?

Reply:
As long as we using the Image component component to render the image, we're good. Nextjs Image has several built-in features that we want to include.